internal/namespaces/baremetal/v1alpha1/custom_server.go

```diff
@@ -0,0 +1,99 @@
+package baremetal
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	serverActionTimeout = 20 * time.Minute
+)
+
+type baremetalActionRequest struct {
+	Zone     scw.Zone
+	ServerID string
+}
+
+var serverActionArgSpecs = core.ArgSpecs{
+	{
+		Name:     "server-id",
+		Short:    `ID of the server affected by the action.`,
+		Required: true,
+	},
+	core.ZoneArgSpec(),
+}
+
+func waitForServerFunc() core.WaitFunc {
+	return func(ctx context.Context, argsI, _ interface{}) (interface{}, error) {
+		return baremetal.NewAPI(core.ExtractClient(ctx)).WaitForServer(&baremetal.WaitForServerRequest{
+			ServerID: argsI.(*baremetalActionRequest).ServerID,
+			Zone:     argsI.(*baremetalActionRequest).Zone,
+			Timeout:  serverActionTimeout,
+		})
+	}
+}
+
+func serverWaitCommand() *core.Command {
+	return &core.Command{
+		Short:     `Wait for server to reach a stable state`,
```

Comment:
```suggestion
		Short:     `Wait for a server to reach a stable state`,
```

Use same text as in the example below ?